packages/url_launcher/url_launcher/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,31 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.urllauncherexample">
 
+  <!-- The INTERNET permission is required for development. Specifically,
+       flutter needs it to communicate with the running application
+       to allow setting breakpoints, to provide hot reload, etc.
+  -->
+  <uses-permission android:name="android.permission.INTERNET"/>
+
+  <!-- Provide required visibility configuration for API level 30 and above -->
+  <queries>
+    <intent>
+      <action android:name="android.intent.action.VIEW" />
+      <data android:scheme="https" />
+    </intent>
+    <!-- Uncomment below for call or email launches with API level 30 and above -->
```

Comment:
It seems odd to put this commented-out section in the example, since the example isn't something we expect people to duplicate directly to make their own app from. The README would be a better place for this part of the change.

Reply:
I'm also not big fan of commented out lingering sections in code. I'll introduce it into the README, devoting a paragraph mashing together what the code documentation refers to and this element. Just to confirm we are on the same page: I'd leave the `<queries>` in the Manifest but not the commented section in that element?

Reply:
Ok, I removed two intents, but then I re-added the intent for `tel` because I realized the example demos phone call. I added an Android section to the README, I wonder if it should below the example (which comes after the iOS) or the Android should be in between the Example and the iOS.
Besides that I think some of my commits went in with my other email addresses I don't have CLA with? Not sure what's going on but it's all me.

Reply:
I'm not sure how to bring back the CLA tag because although I have multiple emails I only have one GitHub account to comment with.